fast_trade/build_data_frame.py

```diff
@@ -70,6 +70,13 @@ def prepare_df(df: pd.DataFrame, backtest: dict):
             1 - float(trailing_stop_loss)
         )
 
+    # take_profit = backtest.get("take_profit", 0)
```

Comment:
we can remove dead commented out code as we code, it keeps it much cleaner and easier to work in :)